docker/score-client-init/download-analysis.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+token=${1:-f69b726d-d40f-4261-b105-1ec7e6bf04d5}
+study-id=${2:-TEST-CA}
+analysis-id=${3:-f69b726d-d40f-4261-b105-1ec7e6bf04d5}
```

Comment:
cant use hyphens here, change them to underscores

Reply:
I don't understand this comment.

Reply:
when you run this script, there is an error becuase `study-id` is not a valid variable name. it should be `study_id`

Reply:
Thanks... for some reason, all I was seeing was the :- and - in TEST-CA and in the analysis id...

Reply:
Need more☕️